### PR TITLE
Add acos SparkSQL function

### PR DESCRIPTION
**Pull request by wypb**
Spark support [acos](https://spark.apache.org/docs/latest/api/sql/#acos) since 1.4.0, this pull request add it to Velox.

### CODE REVIEW

**Comment by wypb** [comment]
HI @mbasmanova could you help me review this new PR, thank you.

File: velox/docs/functions/spark/math.rst [hunk 1]
```diff
@@ -6,6 +6,9 @@ Mathematical Functions
 
     Returns the absolute value of ``x``.
 
+.. spark:function:: acos(x) -> double
+    Returns the inverse cosine (a.k.a. arc cosine) of ``x``.
```
Comment (mbasmanova) [inline velox/docs/functions/spark/math.rst:10]: Would you check the generated docs? I feel you may need to add an empty line before this.

File: velox/functions/sparksql/tests/ArithmeticTest.cpp [hunk 2]
```diff
@@ -183,6 +183,19 @@ TEST_F(ArithmeticTest, Divide) {
   EXPECT_TRUE(std::isnan(divide(kInf, -kInf).value_or(0)));
 }
 
+TEST_F(ArithmeticTest, acos) {
```
Comment (mbasmanova) [inline velox/functions/sparksql/tests/ArithmeticTest.cpp:186]: Is this test needed given that you are reusing the implementation?

File: velox/docs/functions/spark/math.rst (hunk 1 above)
Comment (wypb) [reply velox/docs/functions/spark/math.rst:10]: done

**Pull request closed without merging**